tests/unit/physical_test.py

```diff
@@ -381,7 +381,7 @@ def test_create_physical_fileset_invalid_operating_system(rubrik):
 
     error_message = error.value.args[0]
 
-    assert error_message == "The create_physical_fileset() operating_system argument must be one of the following: ['Linux', 'Windows']."
```

Comment:
Don't need to include `UnixLike` here since we don't support it in the `create_physical_fileset()` function yet.